### DISCUSSION TITLE
Public developers pull data and IP causes associated consultation

### DISCUSSION THREAD

**Discussion by WorkingSilence-L** [amzn/selling-partner-api-models#2815] [General] [answered]
We registered as a public developer, wanted to become a service provider, and authorized some seller stores, but we only have one IP address. When obtaining order data from authorized stores, I use the same IP to extract store data from all sellers. Will this cause these stores to be linked? Will this happen?

**Comment by stefnats** [comment, marked as answer]
They won't be linked, just because one single server pulls the data from the API.
